### PR TITLE
Make that Origin country is taken from the config

### PR DESCRIPTION
**Pull request by matishadow**

### CODE REVIEW

File: DIGNDB.APP.SmitteStop.Jobs/HangfireStartup.cs [hunk 1]
```diff
@@ -43,26 +26,24 @@ public HangfireStartup(IWebHostEnvironment env, IConfiguration configuration)
...
+            loggerFactory.AddFile(hangfireConfig.LogsPath);
 
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
             app.UseHangfireDashboard();
 
-            app.UseHttpsRedirection();
+            // app.UseHttpsRedirection(); // TODO test
```
Comment (matishadow) [inline DIGNDB.APP.SmitteStop.Jobs/HangfireStartup.cs]: I've tested on local machine and `http` works with and without this.
So I guess you can remove this comment.

File: DIGNDB.App.SmitteStop.Core/Services/PackageBuilderService.cs [hunk 2]
```diff
@@ -84,9 +83,9 @@ public List<byte[]> BuildPackageContentV2(DateTime startDate, ZipFileOrigin orig
             return streamBytesList;
         }
 
-        private IList<TemporaryExposureKey> GetNextKeysBatch(ZipFileOrigin origin, DateTime startDate, int numberOfRecordsToSkip)
+        private IList<TemporaryExposureKey> GetNextKeysBatch(string originPostfix, DateTime startDate, int numberOfRecordsToSkip)
         {
-            if (origin == ZipFileOrigin.All)
+            if (originPostfix.ToLower() == ZipFileOrigin.All.ToString().ToLower())
```
Comment (matishadow) [inline DIGNDB.App.SmitteStop.Core/Services/PackageBuilderService.cs]: You could use
```
string.Equals(originPostfix, ZipFileOrigin.All.ToString(), StringComparison.CurrentCultureIgnoreCase)
```
To get rid of two `.ToLower()` invocations.

File: DIGNDB.APP.SmitteStop.Jobs/HangfireStartup.cs (hunk 1 above)
Comment (netcompany-martinropcke) [reply DIGNDB.APP.SmitteStop.Jobs/HangfireStartup.cs]: While we're at it do we need the following in startup or can the be deleted?!

app.UseRouting();
app.UseAuthorization();
app.UseEndpoints(endpoints =>
{
    endpoints.MapControllers();
});

Comment (matishadow) [reply DIGNDB.APP.SmitteStop.Jobs/HangfireStartup.cs]: We do need these lines because Hangfire provides Web dashboard.

**Review by flamaniac — APPROVED** [review]

**Review by netcompany-martinropcke — APPROVED** [review]

**Pull request merged**